IQSS/dataverse
# bump version to 5.12

**What this PR does / why we need it**:

Bumps the version to 5.12.

**Which issue(s) this PR closes**:

Closes #8980

**Special notes for your reviewer**:

None.

**Suggestions on how to test this**:

I checked these already but:

- make sure you see 5.12 at http://localhost:8080/api/info/version
- check for 5.12 at top of html/index.html
  - preview at https://dataverse-guide--9004.org.readthedocs.build/en/9004/index.html
- check new link at build/html/versions.html
  - preview at https://dataverse-guide--9004.org.readthedocs.build/en/9004/versions.html

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

No.

**Is there a release notes update needed for this change?**:

No.

**Additional documentation**:

None.
